lua/lazyvim/plugins/ui.lua

```diff
@@ -306,7 +306,15 @@ return {
   {
     "echasnovski/mini.icons",
     lazy = true,
-    opts = {},
+    opts_extend = { "directory", "extension", "file", "filetype", "lsp", "os" },
```

Comment:
Sorry, naive question here. Is this really needed? I thought `opts_extend` was to extend list-like tables. Aren't the keys defined here key/value-like tables? I just want to understand and am not insinuating anything.

Reply:
Yeah, you're right I should have omited `opts_extend` in this context.